app/controllers/main_controller.rb

```diff
@@ -1,5 +1,7 @@
+require_relative '..lib/string_segmenter.rb'
```

Comment:
I know Alex is helping you, but would changing this to `'../lib/string_segmenter.rb'` make a difference?